src/main/java/seedu/agendum/ui/HelpWindow.java

```diff
@@ -110,29 +116,63 @@ private void configure(){
         Scene scene = new Scene(mainPane);
         //Null passed as the parent stage to make it non-modal.
         dialogStage = createDialogStage(TITLE, null, scene);
-        dialogStage.setWidth(WIDTH);
+//        dialogStage.setWidth(WIDTH);
         dialogStage.setHeight(HEIGHT);
         dialogStage.setResizable(false);
         
         scene.setFill(Color.TRANSPARENT);
         dialogStage.initStyle(StageStyle.TRANSPARENT);
         
         setIcon(dialogStage, ICON);
+        configureDrag();
         loadHelpList();
         
-        handleEscape(scene);
+        handleKeyInput(scene);
+    }
+    
+    private void configureDrag() {
+        mainPane.setOnMousePressed(new EventHandler<MouseEvent>() {
+            @Override
+            public void handle(MouseEvent event) {
+                xOffset = dialogStage.getX() - event.getScreenX();
+                yOffset = dialogStage.getY() - event.getScreenY();
+            }
+        });
+        
+        mainPane.setOnMouseDragged(new EventHandler<MouseEvent>() {
+            @Override
+            public void handle(MouseEvent event) {
+                dialogStage.setX(event.getScreenX() + xOffset);
+                dialogStage.setY(event.getScreenY() + yOffset);
+            }
+        });
```

Comment:
haha because I previously did some special handling on this window to remove the title bar, so we cannot drag it anymore.